fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -15,11 +15,21 @@ def running_help_command?
       def take_off
         before_import_time = Time.now
 
+        # Usually in the fastlane code base we use
```

Comment:
not really deliver related...

Reply:
I was thinking: when people want to add the loading indicator to more areas in fastlane, they're gonna look on how it's done, so they'll look here, but here we'll tell them how to do it right